### DISCUSSION TITLE
Can CRSFforArduino communicate half duplex?

### DISCUSSION THREAD

**Discussion by zhuwei** [ZZ-Cat/CRSFforArduino#63] [Q&A]
When ESP32 is used as TX, it can only communicate in half duplex mode. I would like to use CRSF for Arduino to communicate with TX. Can I use it directly?
PS: I know that communication with RX is full duplex, and CRSF for Arduino can be used directly.

**Comment by ZZ-Cat** [comment]
Could you please clarify this?  
What exactly do you mean, here?

What is it that you're trying to achieve?
